src/scripts/lightgbm_python/train.py

```diff
@@ -49,6 +49,8 @@ def get_arg_parser(parser=None):
         required=True, type=str, help="Training data location (file or dir path)")
     group_i.add_argument("--test",
         required=True, type=str, help="Testing data location (file path)")
+    group_i.add_argument("--construct",
+        required=False, default=True, type=strtobool, help="pre-construct datasets")
```

Comment:
Is "pre-construct dataset" == "lazy initialization" here? Not sure whether "lazy initialize dataset" is more informative help message.

Reply:
definitely, thank you!